workflows/data_pipelines/elasticsearch/mapping_index.py

```diff
@@ -350,6 +350,7 @@ class UniteLegaleMapping(InnerDoc):
     liste_dirigeants = Text(analyzer=annuaire_analyzer)
     liste_elus = Text(analyzer=annuaire_analyzer)
     liste_idcc_unite_legale = Text()
+    liste_finess_juridique = Text()
```

Comment:
I would also rename the `liste_finess` in Etablissement object to `liste_finess_geographique` maybe.

Reply:
This would not change the api response, it's simply for the index in order to be cleaner.
If we decide to later on change that API, it would be gradual I think even though I don't think this field is used that much.